### PR TITLE
Ukrainian Language - "Introduction tutorial" internal linking

### PR DESCRIPTION
**Pull request by butuzov**
This PR related to issue #20316, but issue in particular this PR is internal links weren't interlinked. Some of the translated content ware referenced to the English version.

### CODE REVIEW

File: content/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h2 style="color: #3771e3;">Як розгорнути ваш перший зас
...
                 -->
-                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/docs/tutorials/kubernetes-basics/scale-intro/">5</a> і <a href="/docs/tutorials/kubernetes-basics/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
+                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/uk/docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> і <a href="/uk/docs/tutorials/kubernetes-basics/update/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
```
Comment (MaxymVlasov) [inline content/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:106]: Додавання цих елементів шляху - виправлення чи обдруківка?
```suggestion
                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/uk/docs/tutorials/kubernetes-basics/scale-intro/">5</a> і <a href="/uk/docs/tutorials/kubernetes-basics/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
```

File: content/uk/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h3>Цілі</h3>
...
                 <!--<p>When you created a Deployment in Module <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes created a <b>Pod</b> to host your application instance. A Pod is a Kubernetes abstraction that represents a group of one or more application containers (such as Docker or rkt), and some shared resources for those containers. Those resources include:</p>
                 -->
-                <p>Коли ви створили Deployment у модулі <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes створив <b>Pod</b>, щоб розмістити ваш застосунок. Pod - це абстракція в Kubernetes, що являє собою групу з одного або декількох контейнерів застосунку (як Docker або rkt) і ресурси, спільні для цих контейнерів. До цих ресурсів належать:</p>
+                <p>Коли ви створили Deployment у модулі <a href="/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes створив <b>Pod</b>, щоб розмістити ваш застосунок. Pod - це абстракція в Kubernetes, що являє собою групу з одного або декількох контейнерів застосунку (як Docker або rkt) і ресурси, спільні для цих контейнерів. До цих ресурсів належать:</p>
```
Comment (MaxymVlasov) [inline content/uk/docs/tutorials/kubernetes-basics/explore/explore-intro.html:44]: Аналогічне пиптання

File: content/uk/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 3]
```diff
@@ -151,7 +151,7 @@ <h2 style="color: #3771e3;">Узагальнена схема вузлів</h2>
                 <h2>Діагностика за допомогою kubectl</h2>
                 <!--<p>In Module <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, you used Kubectl command-line interface. You'll continue to use it in Module 3 to get information about deployed applications and their environments. The most common operations can be done with the following kubectl commands:</p>
                 -->
-                <p>У модулі <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a> ви вже використовували інтерфейс командного рядка kubectl. У модулі 3 ви продовжуватимете користуватися ним для отримання інформації про застосунки та оточення, в яких вони розгорнуті. Нижченаведені команди kubectl допоможуть вам виконати наступні поширені дії:</p>
+                <p>У модулі <a href="/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a> ви вже використовували інтерфейс командного рядка kubectl. У модулі 3 ви продовжуватимете користуватися ним для отримання інформації про застосунки та оточення, в яких вони розгорнуті. Нижченаведені команди kubectl допоможуть вам виконати наступні поширені дії:</p>
```
Comment (MaxymVlasov) [inline content/uk/docs/tutorials/kubernetes-basics/explore/explore-intro.html:154]: Те саме

File: content/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 1 above)
Comment (butuzov) [reply content/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:106]: You can take a look at #20316, which covers exactly this issue.

**Pull request merged**